db/seeds.rb

```diff
@@ -87,7 +87,7 @@ def load_test_users
       suburb_file.pos = 0 if suburb_file.eof?
       row = CSV.parse(suburb_file.readline)
 
-      suburb, country, state, latitude, longitude = row[0]
+      suburb, _country, _state, latitude, longitude = row[0]
```

Comment:
Unfortunately, I can't remember where I pulled the suburbs.csv. The purpose of the file was to improve the speed of generating test users. I remember there was a view that shows a number of growers/gardens in bubbles on a map. I think geolocation was normally pulled from external geocoding service when an address was entered in on a user record. Doing this for all test users was causing long wait.
I hope this information helps.